client/src/sections/resources/2024/SeptEmberCenser.tsx

```diff
@@ -0,0 +1,92 @@
+import { ListItem, Text, UnorderedList } from "@chakra-ui/react";
+import { availableAmount, myPrimestat, numericModifier } from "kolmafia";
+import { $item, get } from "libram";
+
+import DynamicItemLinks from "../../../components/DynamicItemLinks";
+import Line from "../../../components/Line";
+import Tile from "../../../components/Tile";
+import { haveUnrestricted } from "../../../util/available";
+import { plural } from "../../../util/text";
+
+const SeptEmberCenser = () => {
+  const septEmberCenser = $item`Sept-Ember Censer`;
+
+  if (!haveUnrestricted(septEmberCenser)) {
+    return null;
+  }
+
+  const bembershoot = $item`bembershoot`;
+  const mmmBrrMouthwash = $item`Mmm-brr! brand mouthwash`;
+  const emberingHunk = $item`embering hunk`;
+  const septEmbers = get("availableSeptEmbers", 0);
+  const coldResistance = numericModifier("cold resistance");
+  const mainstatGain = Math.floor(
+    7 *
+      coldResistance ** 1.7 *
+      (1.0 + numericModifier(`${myPrimestat()} Experience Percent`) / 100.0),
+  );
+  const bembershootCount = availableAmount(bembershoot);
+  const mouthwashCount = availableAmount(mmmBrrMouthwash);
+  const hunkCount = availableAmount(emberingHunk);
+  const hulkFought = get("_emberingHulkFought", false);
+  const structureUsed = get("_structuralEmberUsed", false);
+
+  return (
+    <Tile
+      header="Sept-Ember Censer"
+      href="shop.php?whichshop=september"
+      imageUrl="/images/itemimages/embercenser.gif"
```

Comment:
Using linkedContent here makes Yorick add a use link that doesn't work properly since Censer works as a shop. We could add some special handling for shops in DynamicLinks but I decided to leave that out for now.